docs/_docs/permalinks.md

```diff
@@ -203,26 +232,11 @@ Jekyll also provides the following built-in styles for convenience.
 </table>
 </div>
 
-## Pages and collections
-
-The `permalink` configuration setting specifies the permalink style used for
-posts. Pages and collections each have their own default permalink style; the
-default style for pages is `/:path/:basename` and the default for collections is
-`/:collection/:path`.
+Rather than typing `permalink: /:categories/:year/:month/:day/:title/`, you can just type `permalink: date`. 
 
-These styles are modified to match the suffix style specified in the post
-permalink setting. For example, a permalink style of `pretty`, which contains a
-trailing slash, will update page permalinks to also contain a trailing slash:
-`/:path/:basename/`. A permalink style of `date`, which contains a trailing
-file extension, will update page permalinks to also contain a file extension:
-`/:path/:basename:output_ext`. The same is true for any custom permalink style.
+## Permalink style examples with posts
 
-The permalink for an individual page or collection document can always be
-overridden in the [YAML Front Matter](../frontmatter/) for the page or document.
-Additionally, permalinks for a given collection can be customized [in the
-collections configuration](../collections/).
-
-## Permalink style examples
+Here are a few examples to clarify how permalink styles get applied with posts.
```

Comment:
unless we want to capitalize posts throughout the documentation, i wouldn't capitalize it here. it would inconsistent, wouldn't it?